src/types/resolveStatements.ts

```diff
@@ -175,14 +175,14 @@ function processStatements(
             const variableType = resolveTypeRef(ctx, s.type);
             if (!isAssignable(expressionType, variableType)) {
                 throwError(
-                    `Type mismatch: ${printTypeRef(expressionType)} is not assignable to ${printTypeRef(variableType)}`,
+                    `Type mismatch: "${printTypeRef(expressionType)}" is not assignable to "${printTypeRef(variableType)}"`,
                     s.ref,
                 );
             }
 
             // Add variable to statement context
             if (sctx.vars.has(s.name)) {
-                throwError(`Variable already exists: ${s.name}`, s.ref);
+                throwError(`Variable already exists: "${s.name}"`, s.ref);
```

Comment:
Let's change this for consistency reasons to this:
```suggestion
                throwError(`Variable "${s.name}" already exists`, s.ref);
```